File: legacy_python/models/prenda.py
from extensions import db

class Prenda(db.Model):
    __tablename__ = "prendas"

    id = db.Column(db.Integer, primary_key=True)
    nombre = db.Column(db.String(100), nullable=False)
    tipo = db.Column(db.String(50), nullable=False)
    precio = db.Column(db.Float, nullable=False)

    def __repr__(self):
        return f"<Prenda {self.nombre}>"

File: app/Models/Prenda.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Factories\HasFactory;
use Illuminate\Database\Eloquent\Model;

class Prenda extends Model
{
    use HasFactory;

    protected $table = 'prendas';
    protected $fillable = ['nombre', 'tipo', 'precio'];

    public function producciones()
    {
        return $this->hasMany(Produccion::class);
    }
}
